api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -322,15 +370,15 @@ private void assertChildTurnsOneEmailPersonalisationMap(PaymentCycle currentCycl
                 .contains(formatVoucherAmount(4)); // child is turning one in the next payment cycle, so one voucher per week going forward
     }
 
-    private void assertChildTurnsFourEmailPersonalisationMap(PaymentCycle currentCycle, Map childTurnsOnePersonalisationMap) {
-        assertCommonEmailFields(currentCycle, childTurnsOnePersonalisationMap);
-        assertThat(childTurnsOnePersonalisationMap.get(CHILDREN_UNDER_1_PAYMENT.getTemplateKeyName())).asString()
+    private void assertChildTurnsFourEmailPersonalisationMap(PaymentCycle currentCycle, Map childTurnsFourPersonalisationMap) {
```

Comment:
Trivial, but can we not just call it `personalisationMap`?

Reply:
Yep, will do in the next PR.